README.md

```diff
@@ -293,156 +195,34 @@ Your PDS member (or PS) is uploaded.
-2. Open the **Unix System Services (USS)** bar.
-3. Click the **Add Session** button on the right of the **Unix System Services (USS)** explorer bar.
-4. Select the profile that you want to add to the view as illustrated by the following screen.
-
-![View Profiles](docs/images/ZE-profile2.gif?raw=true "View Profiles")
-<br /><br />
-
----
-
-### JOBS
-
-You can use the following functionalities when interacting with Jobs:
-
-* **View a job**: You can view multiple jobs simultaneously.
-* **Download spool content**: You can download spool content on your computer.
-
-#### View a job
-
-1. Navigate to the explorer tree.
-2. Open the **JOBS** bar.
-3. Select a directory with JCL files.
-4. Right-click on the JCL you want to view, and click **Get JCL**.
-
-![View JOB](docs/images/ZE-jobs-get-jcl.gif?raw=true "View JOB")
-<br /><br />
+## Usage tips
 
-#### Download spool content
+* Use the **Add Favorite** feature to permanently store chosen data sets, USS files, and jobs in the **Favorites** folder. Right-click on a data set, USS file or jobs and select **Add Favorite**.
 
-1. Navigate to the explorer tree.
-2. Open the **JOBS** bar.
-3. Select a directory with JCL files.
-4. Click the **Download** icon next to a folder with the spool content.
-5. Save the file on your computer.
+* **Syntax Highlighting:** Zowe Explorer supports syntax highlighting for data sets. Fox example, you can use such extensions as [COBOL Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.cobol-language-support) or [HLASM Language Support](https://marketplace.visualstudio.com/items?itemName=broadcomMFD.hlasm-language-support).
 
-![Download Spool](docs/images/ZE-jobs-download-spool.gif?raw=true "Download Spool")
-<br /><br />
+* **Edit a profile**: Click the **pencil** icon next to the **magnifying glass** icon in the explorer tree, and modify the information inside your profile.
 
----
+* **Delete a profile**: Right-click a chosen profile and select **Delete Profile** to permanently delete the profile. The functionality deletes a profile from your `.zowe` folder.
 
-### Extras
+* **Hide a profile**: You can hide a profile from the profile tree by right-clicking the profile and selecting the **Hide Profile** option.
 
-#### Issue MVS commands
+For information how to configure Zowe Explorer, see [Zowe Explorer Configuration guidelines](https://docs.zowe.org/stable/user-guide/ze-install.html#configuration).
 
-Zowe Explorer also enables you to issue MVS command. You can issue such commands as Allocate or Exec against a profile.
+## Extending Zowe Explorer
 
-1. Press the **F1** key on your keyboard.
-2. Select the **Zowe:Issue MVS Command** option.
-3. Select your profile.
-4. Issue a MVS command.
+You can extend the possibilities of Zowe Explorer by creating you own extensions. For more information, see [Extensions for Zowe Explorer](https://github.com/zowe/vscode-extension-for-zowe/blob/master/docs/README-Extending.md).
```

Comment:
"You can add new functionalities to Zowe Explorer by creating you**r** own extension", or something like that.

Reply:
Maybe "You can extend the functionalities of Zowe Explorer by creating your own extension"? To me "add new functionalities to Zowe Explorer" sounds like an invitation to work with the Zowe Explorer codebase.

Reply:
I was trying to avoid "you can extend...by creating your own extension"--that's a bit redundant.

Reply:
Fair enough